src/models/Request.js

```diff
@@ -0,0 +1,15 @@
+export default (sequelize, DataTypes) => {
+  const Request = sequelize.define('requests', {
+    status: {
+      type: DataTypes.ENUM('pending', 'rejected', 'approved'),
+      allowNull: false,
+      defaultValue: 'pending',
+    },
+  }, {});
+  Request.associate = (models) => {
+    Request.hasMany(models.comments, { as: 'requestComments', foreignKey: 'requestId' });
+    Request.belongsTo(models.users, { as: 'requestUser', foreignKey: 'userId' });
+    Request.hasMany(models.trips, { as: 'requestTrips', foreignKey: 'requestId' });
```

Comment:
Why are you using `requestId` as the foreign key for a trip

Reply:
Trips are tied to a request and can't be created without them

Reply:
I need you to explain how the `hasMany` relationship works